src/test/java/guitests/SortTest.java

```diff
@@ -50,5 +50,19 @@ public void sortTest () {
         press(KeyCode.SHIFT).press(KeyCode.SEMICOLON).release(KeyCode.SEMICOLON).release(KeyCode.SHIFT);
         type("-date");
         push(KeyCode.ENTER);
+        // Issue state (open)
+        doubleClick("#dummy/dummy_col0_filterTextField");
+        doubleClick("#dummy/dummy_col0_filterTextField");
+        type("sort");
+        press(KeyCode.SHIFT).press(KeyCode.SEMICOLON).release(KeyCode.SEMICOLON).release(KeyCode.SHIFT);
+        type("status");
```

Comment:
special chars such as `:` or `_` wont work for `type()`. I'll create a separate issue for this. The whole class needs to be changed.